src/03-control-flow.js

```diff
@@ -109,7 +149,22 @@ function aroundTheWorldGreeting(language) {}
  *  aroundTheWorldGreetingWithSwitch();
  *  //> "Hello World"
  */
-function aroundTheWorldGreetingWithSwitch(language) {}
+function aroundTheWorldGreetingWithSwitch(language = 'english') {
+  switch (language) {
+    case 'english':
```

Comment:
Lets watch the spacing here - using switch statements in VSCode sometimes will begin to indent too much.  Ideally we want all our `case` statements to line up vertically.